hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/GatewayServiceEntityDao.java

```diff
@@ -52,31 +58,45 @@
                 grpcChannelRegistry.forAddress(
                     serviceConfig.getGatewayServiceHost(), serviceConfig.getGatewayServicePort()))
             .withCallCredentials(credentials);
+
+    this.labelConfigServiceStub =
```

Comment:
commented above - delegate to the other schema, rather than implementing a separate dependency to the label service.